BibBuddy/app/src/main/java/de/bibbuddy/LibraryFragment.java

```diff
@@ -102,6 +103,63 @@ public boolean onOptionsItemSelected(MenuItem item) {
     return super.onOptionsItemSelected(item);
   }
 
+  private void handleBackupLibrary() {
+    setStoragePermission();
+  }
+
+  private void setStoragePermission() {
+    if (ContextCompat.checkSelfPermission(getContext(),
+              Manifest.permission.WRITE_EXTERNAL_STORAGE) != PackageManager.PERMISSION_GRANTED) {
+      requestStoragePermission();
+    }
+  }
+
+  private void requestStoragePermission() {
+    if (ActivityCompat.shouldShowRequestPermissionRationale(getActivity(),
+            Manifest.permission.WRITE_EXTERNAL_STORAGE)) {
+      showRequestPermissionDialog();
+    } else {
+      requestPermissions(new String[] {Manifest.permission.WRITE_EXTERNAL_STORAGE},
+              STORAGE_PERMISSION_CODE);
+    }
+  }
+
+  private void showRequestPermissionDialog() {
+    new AlertDialog.Builder(getContext())
+        .setTitle("Zugriff erforderlich")
+        .setMessage("Zugriff auf den Gerätespeicher erforderlich, "
+            + "um Dateien aus dieser App zu exportieren.")
+        .setPositiveButton("OK", new DialogInterface.OnClickListener() {
+          @Override
+          public void onClick(DialogInterface dialog, int which) {
+            ActivityCompat.requestPermissions(getActivity(),
+                new String[] {Manifest.permission.WRITE_EXTERNAL_STORAGE},
+                STORAGE_PERMISSION_CODE);
+          }
+        })
+        .setNegativeButton("Abbrechen", new DialogInterface.OnClickListener() {
```

Comment:
Please add a newline for better readability. Also don't use a hard coded String. Add the String to strings.xml.